lib/decompress.ps1

```diff
@@ -16,7 +16,25 @@ function Test-7zipRequirement {
             return ($URL | Where-Object { Test-7zipRequirement -File $_ }).Count -gt 0
         }
     } else {
-        return $File -match '\.((gz)|(tar)|(tgz)|(lzma)|(bz)|(bz2)|(7z)|(rar)|(iso)|(xz)|(lzh)|(nupkg))$'
+        return $File -match '\.((gz)|(tar)|(tgz)|(lzma)|(bz)|(bz2)|(7z)|(rar)|(iso)|(xz)|(lzh)|(nupkg)|(tar.zst))$'
```

Comment:
Why do you handle this in 7zip when it is not supported?

In general do not touch anything related to 7zip extraction

Reply:
zstd can not handle tar zrchives. should I add tar as dependency?

Reply:
So your current flow is this:

1. Install manifest which has `tar.zst` as extension
1. Add **ONLY** clean 7zip as dependency as it comfort `Test-7zipRequirement`
1. Call `Expand-ZipArchive` (which is not aware that zstd dependency is missing in the most cases)
1. Throw error to users' face that he is trying to bind null as parameter to Get-HelperPath as he is missing dependency to zstd.
1. Abort

Reply:
For me it looks not bad:
```
>scoop install openssh
Installing 'zstd' (1.4.5) [64bit]
Loading zstd-v1.4.5-win64.zip from cache
Checking hash of zstd-v1.4.5-win64.zip ... ok.
Extracting zstd-v1.4.5-win64.zip ... done.
Linking D:\scoop\apps\zstd\current => D:\scoop\apps\zstd\1.4.5
Creating shim for 'zstd'.
'zstd' (1.4.5) was installed successfully!
Installing 'openssh' (8.3p1-1) [64bit]
Loading bash-4.4.023-2-x86_64.pkg.tar.xz from cache
Checking hash of bash-4.4.023-2-x86_64.pkg.tar.xz ... ok.
Loading gcc-libs-9.3.0-1-x86_64.pkg.tar.xz from cache
Checking hash of gcc-libs-9.3.0-1-x86_64.pkg.tar.xz ... ok.
Loading heimdal-7.7.0-2-x86_64.pkg.tar.zst from cache
Checking hash of heimdal-7.7.0-2-x86_64.pkg.tar.zst ... ok.
Loading heimdal-libs-7.7.0-2-x86_64.pkg.tar.zst from cache
Checking hash of heimdal-libs-7.7.0-2-x86_64.pkg.tar.zst ... ok.
Loading libcrypt-2.1-2-x86_64.pkg.tar.xz from cache
Checking hash of libcrypt-2.1-2-x86_64.pkg.tar.xz ... ok.
Loading libdb-5.3.28-3-x86_64.pkg.tar.zst from cache
Checking hash of libdb-5.3.28-3-x86_64.pkg.tar.zst ... ok.
Loading libedit-20191231_3.1-1-x86_64.pkg.tar.xz from cache
Checking hash of libedit-20191231_3.1-1-x86_64.pkg.tar.xz ... ok.
Loading libopenssl-1.1.1.g-3-x86_64.pkg.tar.zst from cache
Checking hash of libopenssl-1.1.1.g-3-x86_64.pkg.tar.zst ... ok.
Loading libreadline-8.0.004-1-x86_64.pkg.tar.xz from cache
Checking hash of libreadline-8.0.004-1-x86_64.pkg.tar.xz ... ok.
Loading libsqlite-3.33.0-2-x86_64.pkg.tar.zst from cache
Checking hash of libsqlite-3.33.0-2-x86_64.pkg.tar.zst ... ok.
Loading msys2-runtime-3.1.7-2-x86_64.pkg.tar.xz from cache
Checking hash of msys2-runtime-3.1.7-2-x86_64.pkg.tar.xz ... ok.
Loading ncurses-6.2-1-x86_64.pkg.tar.xz from cache
Checking hash of ncurses-6.2-1-x86_64.pkg.tar.xz ... ok.
Loading openssh-8.3p1-1-x86_64.pkg.tar.zst from cache
Checking hash of openssh-8.3p1-1-x86_64.pkg.tar.zst ... ok.
Loading openssl-1.1.1.g-3-x86_64.pkg.tar.zst from cache
Checking hash of openssl-1.1.1.g-3-x86_64.pkg.tar.zst ... ok.
Loading tcl-8.6.10-1-x86_64.pkg.tar.xz from cache
Checking hash of tcl-8.6.10-1-x86_64.pkg.tar.xz ... ok.
Loading zlib-1.2.11-1-x86_64.pkg.tar.xz from cache
Checking hash of zlib-1.2.11-1-x86_64.pkg.tar.xz ... ok.
Extracting bash-4.4.023-2-x86_64.pkg.tar.xz ... done.
Extracting gcc-libs-9.3.0-1-x86_64.pkg.tar.xz ... done.
Extracting heimdal-7.7.0-2-x86_64.pkg.tar.zst ... done.
Extracting heimdal-libs-7.7.0-2-x86_64.pkg.tar.zst ... done.
Extracting libcrypt-2.1-2-x86_64.pkg.tar.xz ... done.
Extracting libdb-5.3.28-3-x86_64.pkg.tar.zst ... done.
Extracting libedit-20191231_3.1-1-x86_64.pkg.tar.xz ... done.
Extracting libopenssl-1.1.1.g-3-x86_64.pkg.tar.zst ... done.
Extracting libreadline-8.0.004-1-x86_64.pkg.tar.xz ... done.
Extracting libsqlite-3.33.0-2-x86_64.pkg.tar.zst ... done.
Extracting msys2-runtime-3.1.7-2-x86_64.pkg.tar.xz ... done.
Extracting ncurses-6.2-1-x86_64.pkg.tar.xz ... done.
Extracting openssh-8.3p1-1-x86_64.pkg.tar.zst ... done.
Extracting openssl-1.1.1.g-3-x86_64.pkg.tar.zst ... done.
Extracting tcl-8.6.10-1-x86_64.pkg.tar.xz ... done.
Extracting zlib-1.2.11-1-x86_64.pkg.tar.xz ... done.
Running pre-install script...


    Directory: D:\scoop\apps\openssh\8.3p1-1


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        10/14/2020   5:31 PM                tmp
d-----        10/14/2020   5:31 PM                home


    Directory: D:\scoop\apps\openssh\8.3p1-1\home


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        10/14/2020   5:31 PM                Werner


    Directory: C:\Users\Werner


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----         10/3/2020  10:25 PM                .ssh
Junction created for D:\scoop\apps\openssh\8.3p1-1\home\Werner\.ssh <<===>> C:\Users\Werner\.ssh
Linking D:\scoop\apps\openssh\current => D:\scoop\apps\openssh\8.3p1-1
Creating shim for 'findssl.sh'.
Creating shim for 'scp'.
Creating shim for 'sftp'.
Creating shim for 'ssh'.
Creating shim for 'ssh-add'.
Creating shim for 'ssh-agent'.
Creating shim for 'ssh-copy-id'.
Creating shim for 'ssh-keygen'.
Creating shim for 'ssh-keyscan'.
Creating shim for 'sshd'.
'openssh' (8.3p1-1) was installed successfully!

```